zap/src/main/resources/org/zaproxy/zap/resources/xml/report.html.xsl

```diff
@@ -119,6 +122,44 @@ ZAP Scanning Report
   </tr>
 </table>
 <div class="spacer-lg"></div>
+
+<h3>Alerts</h3>
+<table width="75%" class="alerts">
+  <tr bgcolor="#666666"> 
+    <th width="60%" height="24">Name</th>
+    <th width="20%" align="center">Risk Level</th>
+    <th width="20%" align="center">Number of Instances</th>
+  </tr>
+  <xsl:key name="alerts-by-name-risk" match="alertitem" use="concat(name, ' ', riskcode)"/>
+  <xsl:for-each select="descendant::alertitem[count(. | key('alerts-by-name-risk', concat(name, ' ', riskcode))[1]) = 1]">
+    <xsl:sort order="descending" data-type="number" select="riskcode"/>
+    <xsl:sort order="descending" data-type="number" select="confidence"/>
```

Comment:
I'd suggest removing this since the confidence is no longer shown (otherwise it's not obvious what the order is).

Reply:
Okay, I'll do that.